fastapi_azure_auth/auth.py

```diff
@@ -189,7 +189,8 @@ async def __call__(self, request: Request, security_scopes: SecurityScopes) -> O
                         options=options,
                     )
                     # Attach the user to the request. Can be accessed through `request.state.user`
-                    user: User = User(**token | {'claims': token, 'access_token': access_token})
```

Comment:
Please change this to:
```python
user: User = User({**token, 'claims': token, 'access_token': access_token})
```

Reply:
Hey Per,
Thanks for the one liner, but pydantic model can't accept dictionary directly but an unpacked dictionary(**dictionary), that's why it is giving error. Can I go with this -

```user: User = User(**token, claims = token, access_token = access_token)```